packages/website/ts/components/eth_weth_conversion_button.tsx

```diff
@@ -12,12 +12,15 @@ import {errorReporter} from 'ts/utils/error_reporter';
 import {utils} from 'ts/utils/utils';
 
 interface EthWethConversionButtonProps {
+    direction: Side;
     ethToken: Token;
     ethTokenState: TokenState;
     dispatcher: Dispatcher;
     blockchain: Blockchain;
     userEtherBalance: BigNumber;
-    onError: () => void;
+    isOutdatedWrappedEther: boolean;
+    onConversionSuccessful?: () => void;
```

Comment:
IfExists

Reply:
I don't think it makes sense to add the `IfExists` prefix to prop names. You might or might not define it when instantiating the component. I'll add a defaultProp value instead.